processes.py

```diff
@@ -131,10 +131,20 @@ def process_to_dict(process):
        'TAG issues addressed',
        'Three LGTMs',
        'Updated vendor signals',
-       'Finalized target milestone',
+       'Updated target milestone',
       ],
       [('Draft Intent to Ship email', INTENT_EMAIL_URL)],
       models.INTENT_IMPLEMENT_SHIP, models.INTENT_SHIP),
+
+  ProcessStage(
+      'Ship',
+      'Update milestones and other information when the feature '
+      'actually ships.',
```

Comment:
```suggestion
      'ships.',
```

Reply:
Maybe "when the feature has actually shipped" is the right phrase.  I get why Joe doesn't like "actually" here, but I worry we're not being directive enough without it?

Reply:
I'll keep what I have for this CL just so that I can land this for a UX test today, but I am happy to make whatever wording change we agree on in an upcoming CL.